### PR TITLE
pinot version upgrade

### PR DESCRIPTION
**Pull request by laxmanchekka**

### CODE REVIEW

File: .trivyignore [hunk 1]
```diff
@@ -1,5 +1,11 @@
-# org.yaml:snakeyaml (from upstream opensource dependency)
+# org.yaml:snakeyaml (from upstream opensource shaded dependency. will be taken care during upgrade.)
 CVE-2022-1471 exp:2023-09-30
 
-# net.minidev:json-smart (from upstream opensource dependency)
+# net.minidev:json-smart (from upstream opensource shaded dependency. will be taken care during upgrade.)
 CVE-2023-1370 exp:2023-09-30
+
+# com.google.guava:guava (from upstream opensource shaded dependency. will be taken care during upgrade.)
+CVE-2023-2976
```
Comment (ravisingal) [inline .trivyignore:8]: add expiry as well.

File: .trivyignore [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+# org.apache.helix:helix-core (from upstream opensource shaded dependency. will be taken care during upgrade.)
+CVE-2023-38647
```
Comment (ravisingal) [inline .trivyignore:11]: add expiry as well.

File: .trivyignore (hunk 1 above)
Comment (laxmanchekka) [reply .trivyignore]: done

File: .trivyignore (hunk 2 above)
Comment (laxmanchekka) [reply .trivyignore]: done

**Review by ravisingal — APPROVED** [review]

**Pull request merged**